backend/controller/console.go

```diff
@@ -228,11 +246,11 @@ func (c *ConsoleService) StreamLogs(ctx context.Context, req *connect.Request[pb
 	}
 }
 
-func convertModuleCalls(calls []dal.CallEvent) []*pbconsole.Call {
+func convertModuleCalls(calls []dal.CallEvent) []pbconsole.Call {
```

Comment:
These changes don't make any sense. Protobuf always uses pointers, so this just forces the caller to convert them back again?

Reply:
Lemme see if I can make it work the other way. I did this to "fix" my new calls but it felt odd. I might ping ya with the error so you can show me the way :)